docs/topics/installing-cli-tool.adoc

```diff
@@ -46,35 +33,143 @@ When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with th
 [id="installing-using-podman_{context}"]
 == Installing the {CLINameTitle} using Podman
 
+You can install the {CLINameTitle} using `podman pull`.
```

Comment:
This doesn't look correct, you cannot install the CLI with `podman pull`

Reply:
Might just be reference to prior line 56
```
To install using `podman pull`:
```
where eventually a cli is copied off an image using `podman cp`

Reply:
I think we still build kantra cli for windows in the linux container image. So you should be able to podman pull it.